source/iostreams.tex

```diff
@@ -15201,7 +15202,7 @@
 For POSIX-based operating systems,
 \tcode{absolute(p)} is simply \tcode{current_path()/p}.
 For Windows-based operating systems,
-\tcode{absolute} might have the same semantics as \tcode{GetFullPathNameW}.
+\tcode{absolute} can have the same semantics as \tcode{GetFullPathNameW}.
```

Comment:
I don't think we're talking about ability or capability here; rather, we're talking about possibility. Given https://github.com/microsoft/STL/blob/master/stl/inc/filesystem#L3001, I think we can be so bold as to say:
```suggestion
\tcode{absolute} typically has the same semantics as \tcode{GetFullPathNameW}.
```

Reply:
fixed

Reply:
Ahem, there are two implementations that support Windows. But [the other one](https://gcc.gnu.org/git/?p=gcc.git;a=blob;f=libstdc%2B%2B-v3/src/c%2B%2B17/fs_ops.cc;h=2cb26e4605b89b9d5ef89dbcde1dcd7d4ec7ef42;hb=HEAD#l118) also uses `GetFullPathNameW` here.